### DISCUSSION TITLE
113年全大運，跳高全國紀錄刷新

### DISCUSSION THREAD

**Discussion by YOGA012** [1101-GS4719-C/Python-Community-GS4719-C#1976] [General]
就讀虎尾科大動力機械系的葉柏廷，在5/7田徑男子跳高項目技驚四座，以2公尺30的成績，打破大會紀錄及全國紀錄奪得金牌，原全國紀錄是向俊賢2015年高雄全運創的2.29，大會紀錄則是臺體大傅兆玄去年創的2.25，以2.26奪銀的中興大學葉維智也破大會，傅兆玄2.23第三。葉柏廷打破高懸九年全國紀錄時，現場觀眾跟著興奮大叫，當時我也在場邊觀看，真的很興奮，可惜接下來挑戰奧運達標高度2公尺33失敗，接下來六月將舉辦臺灣國際田徑公開賽希望葉柏廷可以挑戰成功，前進奧運。另外也期待傅兆玄以及葉維智的表現，期待這三人未來可以互相激勵，將台灣跳高的水準推向國際。另外偷偷許願本校校友鄭詠仁將來有機會復出，繼續在賽場上發光發熱。
詳細報導：
[https://braveathlete.net/tw/article/yeh,po-ting](url)

**Comment by yubo0oo** [comment]
我覺得她超強
